source/sync/go-to-production/production-checklist.txt

```diff
@@ -25,6 +25,17 @@ Avoid breaking changes
   breaking change you need to terminate and re-enable sync with the updated
   data models.
 
+Use a Dedicated Atlas Cluster
```

Comment:
The wording and capitalization of these best practices are inconsistent; might be nice to update across the board for more parallelism (and is "Atlas Cluster" a capitalized term?)
```suggestion
Use a dedicated Atlas Cluster
```